client/collection_field_description.go

```diff
@@ -38,6 +38,9 @@ type CollectionFieldDescription struct {
 	//
 	// Otherwise will be [None].
 	RelationName immutable.Option[string]
+
+	// DefaultValue contains the default value for this field.
+	DefaultValue any
```

Comment:
todo: Please test and document what happens when this is set on a View - and consider adding a validation rule to prevent that (as it makes no sense to me) (and test that, including on Patch).

Reply:
Okay cheers, I'm happy with that :) - can you please document that the property is ignored for views on the directive, and on this `CollectionFieldDescription` prop so that users know?

Reply:
done